src/main.py

```diff
@@ -4,17 +4,16 @@
 from core.sim import CislunarSim
 
 
-class SimRunner():
+class SimRunner:
     def __init__(self, config_path: Optional[str] = None) -> None:
         config = core.config.make_config(config_path)
```

Comment:
Is this function going to be part of the `Config` class? If so we need to make a `Config` object

Reply:
All of main.py is just psuedocode for now, don't treat this as definite yet. We'll want to be able to instantiate `Config` objects from file, probably JSON.